js/DataSourceAggregator.js

```diff
@@ -265,6 +265,18 @@ var DataSourceAggregator = Base.extend('DataSourceAggregator', {
         return this.hasAggregates() && this.hasGroups();
     },
 
+    /**
+     * @memberOf DataSourceAggregator#
+     * @returns {*|boolean}
+     */
+    isDrillDown: function(event) {
+        var result = this.viewMakesSense();
+        if (result && event) {
+            result = event.dataCell.x === this.treeColumnIndex;
```

Comment:
I don't think "events" should be in here or "dataCell" for that matter. We should just send `columnIndex`.